player/db/rounds/rounds.go

```diff
@@ -1 +1,42 @@
 package roundsdb
+
+import (
+	"context"
+	"database/sql"
+	"github.com/adamhicks/ctrlaltdefeat/player"
+)
+
+func InsertRoundParts(ctx context.Context, dbc *sql.DB, matchId, roundId int, playerId string, rank, p1Part, p2Part, p3Part, p4Part int) (int64, error) {
```

Comment:
you won't actually have a match_id so we'll have to use a player_round id (autoincrementing from the player_rounds table)